src/helpers/comment.ts

```diff
@@ -0,0 +1,73 @@
+import { fromMarkdown } from "mdast-util-from-markdown";
+import { gfmFromMarkdown } from "mdast-util-gfm";
+import { gfm } from "micromark-extension-gfm";
+
+export const MarkdownItem = {
+  Paragraph: "paragraph",
+  List: "list",
+  Link: "link",
+  Text: "text",
+  Code: "code",
+} as const;
+
+const PriceItem: Record<string, number> = {
```

Comment:
It should be moved to the bot's config. Should we create a separate issue for "add incentive config"?

Reply:
Yes these should have configurable values. This variable should be named something like `CommentElementPricingDefaults`

Reply:
I guess that would be better to have a separate config. anyway pushed renaming

Reply:
https://github.com/ubiquity/ubiquibot/pull/305/commits/a964f1fc7f02260fad10b7b577a0fd6090265199